Packs/NutanixHypervisor/Classifiers/classifier-Nutanix_Hypervisor_5_9_9.json

```diff
@@ -0,0 +1,126 @@
+{
+    "brandName": "Nutanix Hypervisor",
+    "custom": true,
+    "id": "NutanixHypervisor-mapper",
```

Comment:
```suggestion
    "id": "NutanixHypervisor",
```

Reply:
@Itay4 The ID of the mapper from 6.0.0 and forward is also NutanixHypervisor-mapper - you want me to change its ID too to NutanixHypervisor?